metadata-ingestion/src/datahub/ingestion/api/auto_work_units/auto_dataset_properties_aspect.py

```diff
@@ -0,0 +1,102 @@
+import dataclasses
+from typing import Dict, Iterable, Optional
+
+from datahub.ingestion.api.workunit import MetadataWorkUnit
+from datahub.metadata.schema_classes import (
+    DatasetPropertiesClass,
+    OperationClass,
+    TimeStampClass,
+)
+from datahub.specific.dataset import DatasetPatchBuilder
+
+
+@dataclasses.dataclass
+class TimestampPair:
+    last_modified: Optional[TimeStampClass]
```

Comment:
is this last modified from dataset props 

can we make the variable naming more clear?

Reply:
I can rename it to `last_modified_dataset_properties_aspect` but it is too long, so I have added comment next to variable names.

Reply:
surely you can come up with a variable name that is both shorter than that and more clear than `last_modified`